data/changelog.txt

```diff
@@ -147,67 +154,82 @@ Monsters with larger eyes see farther than smaller variants.
 Ankle storage items don't affect feet-related mutations anymore.
 Give the default Survivor profession more weather-appropriate clothes for the default start.
 Rebalance anesthesia to be more realistic.
+Eating raw meat does not hurt your morale as much if you have traits such as Hunter, Predator, etc.
+Very-fast-moving zombie animals only attack once per turn.
 
 ## Bugfixes:
 Prevents possible negative damage from explosions.
 Removes scent diffusion over water by implementing NO_SCENT flag.
 Give electric motors equal power in both forward and reverse.
-Fix wield action costing double than it should.
-Fix spells not gaining levels on cast.
-Fix for light sources rendering player invisible at certain ranges.
-Fix crash when NPC gives player equipment.
-Fix "Unable to find and assign mission target" message when accepting mission.
+Fixes wield action costing double than it should.
+Fixes spells not gaining levels on cast.
+Fixes for light sources rendering player invisible at certain ranges.
+Fixes crash when NPC gives player equipment.
+Fixes "Unable to find and assign mission target" message when accepting mission.
 No more floor items when trading with follower NPCs.
-Fix crash by not accessing null basecamp.
+Fixes crash by not accessing null basecamp.
 Helicopters: Allow descending into open air while moving, and disallow roof teleporting.
-Fix cold item rotting in freezer and add more rotting tests.
+Fixes cold item rotting in freezer and add more rotting tests.
 Place cursor at highlighted item name in construction and crafting menus as expected by screen readers.
 Spawn regular ants under regular anthills.
 melee: Make sure that melee weapon accuracy improves hit rates.
 Reduce level of vehicle heater/cooler as its now too much after 1-second turns.
 Wake up player if too cold due to hypothermia.
 Fixes a segfault during initializing crafts.
 fixes segfault when attempting to siphon from vehicle with multiple tanks.
-Fix creatures blocking descend even if they can be displaced.
-Fix OnGetHit effects not triggering if player cannot block.
+Fixes creatures blocking descend even if they can be displaced.
+Fixes OnGetHit effects not triggering if player cannot block.
 Refactor monster spawning code to avoid crash.
 Reset game mode before loading normal saves.
 Minimize UI flickering on curses.
```

Comment:
```suggestion
Fixes creatures blocking descent even if they can be displaced.
Fixes OnGetHit effects not triggering if the player cannot block.
Refactors the monster spawning code to avoid crashes.
Resets the game mode before loading normal saves.
Minimizes UI flickering in curses mode.
```